priiloader/hacks_hash.ini

```diff
@@ -1,3 +1,115 @@
f2f63,0x612e6e61,0x732e7769,0x696d6d66,0x692e6465,0x2f636100,0x68747470,0x3a2f2f6e,0x61737769,0x692e7769,0x696d6d66,0x692e6465,0x2f616300,0x68747470,0x733a2f2f,0x6d61696e,0x2e6e6173,0x2e776969,0x6d6d6669,0x2e64652f,0x70700000,0x2c030000,0x40810040,0x8061000c,0x800d9738,0x2c000002,0x41820014,0x906da14c,0x38000002,0x900d9738,0x48000020,0x38830003,0x5484003a,0x88a40000,0x7ca42a14,0x7c0028ac,0x7ca803a6,0x4e800021,0x3860ffff,0x2c030000,0x00000000,0x808da14c,0x28040001,0x408100a0,0x9421ff80,0x38610020,0x3c00801d,0x600024f4,0x7c0803a6,0x4e800021,0x38610020,0x808da14c,0x38a00554,0x90bc00c4,0x3c00801d,0x60002544,0x7c0803a6,0x4e800021,0x38610020,0x38810010,0x3c00801d,0x600025f8,0x7c0803a6,0x4e800021,0x48000018,0x0fff1f07,0x00e638c9,0x49fbeffa,0x79022d3a,0x84ab134f,0x7c6802a6,0x3881000c,0x38a00005,0x7ca903a6,0x84a30004,0x84c40004,0x7c062800,0x40820008,0x4200fff0,0x7c062800,0x38210080,0x808da14c,0x41820008,0x38800000,0x00000000,0x7c651b78,0x88c50000,0x2c060068,0x40a20068,0x88c50001,0x2c060074,0x40a2005c,0x88c50002,0x2c060074,0x40820050,0x88c50003,0x2c060070,0x40820044,0x88c50004,0x2c060073,0x40820038,0x88c50005,0x2c060000,0x40820010,0x38c00000,0x98c50004,0x48000020,0x38e50003,0x39050004,0x8d280001,0x9d270001,0x2c090000,0x4082fff4,0x9d270001,0x88c50000,0x2c06006e,0x40a20118,0x88c50001,0x2c060069,0x40a2010c,0x88c50002,0x2c06006e,0x40820100,0x88c50003,0x2c060074,0x408200f4,0x88c50004,0x2c060065,0x408200e8,0x88c50005,0x2c06006e,0x408200dc,0x88c50006,0x2c060064,0x408200d0,0x88c50007,0x2c06006f,0x408200c4,0x88c50008,0x2c060077,0x408200b8,0x88c50009,0x2c060069,0x408200ac,0x88c5000a,0x2c060066,0x408200a0,0x88c5000b,0x2c060069,0x40820094,0x88c5000c,0x2c06002e,0x40820088,0x88c5000d,0x2c06006e,0x4082007c,0x88c5000e,0x2c060065,0x40820070,0x88c5000f,0x2c060074,0x40820064,0x38e00077,0x98e50000,0x38e00069,0x98e50001,0x98e50002,0x98e50006,0x38e0006d,0x98e50003,0x98e50004,0x38e00066,0x98e50005,0x38e0002e,0x98e50007,0x38e00064,0x98e50008,0x38e00065,0x98e50009,0x39050009,0x3925000f,0x8ce90001,0x9ce80001,0x2c070000,0x4082fff4,0x9ce80001,0x38a50001,0x3d208089,0x6129a66c,0x7c092800,0x41820014,0x3d208089,0x6129a738,0x7c092800,0x40a20008,0x38a50040,0x3d2080a0,0x7c092800,0x4181fe40,0x38a00000,0x3d00801a,0x61086ffc,0x7d0903a6,0x4e800420,0x60000000,0x800d9758,0x906da16c,0x900d9758,0x808da16c,0x60002850,0x808da16c,0x600028a0,0x60002954,0x808da16c,0x3c608000,0x38800052,0x38a0004d,0x38c00043,0x88e30000,0x7c072000,0x4082fb04,0x88e30001,0x7c072800,0x4082faf8,0x88e30002,0x7c073000,0x4082faec,0x8be30003,0x607e5f34,0x4bfffc34,0x7fa802a6,0x3c608027,0x60636054,0x2c1f0045,0x40820008,0x60631d14,0x2c1f004a,0x40820008,0x606359f4,0x2c1f004b,0x4082000c,0x3c608026,0x60633e34,0x389d0004,0x38a0002a,0x7fc803a6,0x4e800021,0x3c608027,0x6063a400,0x2c1f0045,0x40820008,0x606360c0,0x2c1f004a,0x40820008,0x60639da0,0x2c1f004b,0x4082000c,0x3c608026,0x606382b0,0x7c7c1b78,0x389d0030,0x38a0001c,0x7fc803a6,0x4e800021,0x387c0028,0x389d004c,0x38a0001c,0x7fc803a6,0x4e800021,0x387c004c,0x389d0068,0x38a0001f,0x7fc803a6,0x
```

Comment:
what version are these?

Reply:
4609/4610 are Wii Mini (4609 - NTSC-U, 4610 - PAL)

Reply:
That's correct, 4609 and 4610 are the Wii Mini versions. Hasn't been tested at all as I don't want to install Priiloader without Keyboard support on my Wii mini, but it should work.

Reply:
figured as much. i saw the dolphin change to allow versions that was 4 numbers long but i wanted to double check.

wii mini doesn't have wifi doesn't it? does this patch have any use for them then?

Reply:
Wii mini doesn't have WiFi, but you can connect and use a LAN adapter. I've successfully played online using a Wii Mini.

Reply:
ah, ye the lan adaptor, that explains. alright then!